app/models/parliament.rb

```diff
@@ -207,6 +207,13 @@ def start_archiving!(now = Time.current)
     end
   end
 
+  def start_anonymizing!(now = Time.current)
+    if archiving_finished? && Archived::Petition.can_anonymize?
+      time = Date.tomorrow.beginning_of_day
+      Archived::AnonymizePetitionsJob.set(wait_until: time).perform_later(time.iso8601)
```

Comment:
This is fine for now but I think at some point we added native support for timestamp serialization to Active Job so might be able to clean all these `iso8601` methods up in a later PR.